### PR TITLE
Minor Deck 3 comments

### PR DESCRIPTION
**Pull request by DavisVaughan**
Needs to be rendered, like the others

### CODE REVIEW

File: slides/03-what-makes-a-model.qmd [hunk 1]
```diff
@@ -51,7 +51,7 @@ countdown(minutes = 3)
 ::: columns
 ::: {.column width="40%"}
 
--   Choose a model
+-   Choose a [model]{.underline}
```
Comment (DavisVaughan) [inline slides/03-what-makes-a-model.qmd:54]: Underlining seems to show up a lot better to highlight specific parts of a slide

File: slides/03-what-makes-a-model.qmd [hunk 2]
```diff
@@ -150,9 +150,9 @@ decision_tree() %>%
 
 . . .
 
-::: smaller
+<br></br>
+
 All available models are listed at <https://www.tidymodels.org/find/parsnip/>
-:::
```
Comment (DavisVaughan) [inline slides/03-what-makes-a-model.qmd:155]: The smaller thing here doesn't seem to be doing anything, but it does look like we need a little more spacing between the code chunk and this bit of prose so I used a manual break

File: slides/03-what-makes-a-model.qmd [hunk 3]
```diff
@@ -259,23 +259,19 @@ ggplot(data = tree_preds, aes(age, latency)) +
...
 
 . . .
 
--   `fit()` and `predict()` apply to all the estimation steps
+-   They can help organize your work when working with multiple models
 
 . . .
 
--   They can help organize your work when working with multiple models
+-   [Most importantly]{.underline}, a workflow captures the entire modeling process. `fit()` and `predict()` apply to the preprocessing steps in addition to the actual model fit.
```
Comment (DavisVaughan) [inline slides/03-what-makes-a-model.qmd:274]: I felt like it was worth putting this at the end and really hammering it home with some underlining. We could spend a second here talking about why it is important

It is also possible that this slide should come before the workflow diagrams?

Comment (juliasilge) [reply slides/03-what-makes-a-model.qmd:274]: I guess we could move it first, but I tend to think "show" visually first then "tell" with words is a better order.

Comment (DavisVaughan) [reply slides/03-what-makes-a-model.qmd:274]: That's fine! Order is probably less important than the fact that we should just try and mention that "the workflow captures the entire process" multiple times

**Pull request merged**